scss/tech.scss

```diff
@@ -45,3 +45,29 @@
     }
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .tech-grid-container {
+    grid-template-areas: 
+    'left' 
+    'main1'
+    'main2'
+    'main3' 
+    'main4';
+    grid-template-columns: 100%;
+    grid-template-rows: repeat(5, auto);
+    margin: 0 60px 100px;
+
+    .first {
+      border-right: none;
+    }
+    
+    .box {
+      border-bottom: solid $green 3px;
+    }
+
+    .last {
```

Comment:
you can use the last child pseudo-class. https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child